docs/guides/nodejs/nitric-and-pgsql.mdx

```diff
@@ -0,0 +1,222 @@
+---
+description: 'How to work with Nitric SQL Databases using PostgreSQL'
+tags:
+  - Databases & CMS
+languages:
+  - typescript
+  - javascript
+published_at: 2025-03-18
+---
+
+# Nitric SQL Databases with PostgreSQL
+
+This guide demonstrates how to use PostgreSQL with Nitric's [SQL database](/sql) feature. We will be creating a simple to-do app. The finished source can be found [here](https://github.com/nitrictech/examples/tree/main/v1/nitric-pgsql).
+
+<Note>
+  SQL databases are currently in preview and only support PostgreSQL.
+</Note>
+
+## Create a new Nitric project
+
+The first step is to create a new Nitric TypeScript project using the [Nitric CLI](/reference/cli).
+
+```bash
+nitric new todo-app ts-starter
+npm install
+```
+
+## Enabling SQL databases
+
+SQL databases are currently in Preview. To enable this feature in your project, add the following to your `nitric.yaml` file:
+
+```yaml title: nitric.yaml
+preview:
+  - sql-databases
+```
+
+Ensure your CLI is at least `v1.42` and your AWS provider is at least `v1.6.2`.
```

Comment:
We support sql on all clouds on pulumi now so lets update it to mention we support all but will choose AWS for this guide.